scripts/make_toy_data.py

```diff
@@ -0,0 +1,548 @@
+"""
```

Comment:
It's important to preserve history when moving files in Git.  Otherwise, specific changes between revisions, commit messages, and the ability to [bisect](https://git-scm.com/docs/git-bisect) are lost, and with them easy code review and debugging of major latent bugs.  

So instead of:

`mv foo.py ../..`

You'd simply use:

`git mv foo.py ../..`

Please rebase and apply `git mv` accordingly.  We can pair program on this if you'd like.

Reply:
This can be deferred, as we found the rebase isn't trivial.